updates send_file() and send_message()

8go opened the pull request:
Updated 2 API calls send-msg and send-file to work with the latest Dbus API of Jami (as of August 2024)

AmarOk1412 commented:
I'll do the review later. Thx

Note: I saw you contributed to jami-daemon too. Github is not used, https://review.jami.net is the way to send patches

AmarOk1412 commented:
Please squash commit and follow format:

```
type(module): short message (80char max)

long message

Ticket: #XXX
```

where XXX corresponds of the ticket opened for the second comment.


Thank you very much for the contribution, I appreciate it ;)

8go commented:
> Note: I saw you contributed to jami-daemon too. Github is not used, https://review.jami.net/ is the way to send patches

Did submit PR to review.jami.net today. Thank you for pointing it out.

8go committed:
lib: updated functions send_conversation_message() and send_file()

Updated 2 API calls to get sending msgs and sending files
to work with the latest Dbus API of Jami (as of August 2024).

See also https://review.jami.net/c/jami-daemon/+/28889

Ticket: #28889

8go commented:
I hope this works for you.
